adit/dicom_web/utils/wadors_utils.py

```
@@ -95,3 +113,187 @@ async def add_sentinel_when_done(task: asyncio.Task[None]) -> None:
+                temp_path, nifti_output_dir
+            )
+        except (NoValidDicomError, NoSpatialDataError) as e:
+            # The series passed the modality check but still failed conversion.
+            # This is unexpected and worth logging as a warning.
+            logger.warning(f"Series conversion failed unexpectedly: {e}")
+            return
+        except DcmToNiftiConversionError as e:
+            logger.warning(f"Failed to convert DICOM files to NIfTI: {e}")
+            return
+        except Exception as e:
+            logger.error(f"Error during DICOM to NIfTI conversion: {e}")
```

Ruff (0.15.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)